### ISSUE TITLE
Database Schema Installation failed.

### ISSUE THREAD

**Issue by ngreve** [padams/Open-Web-Analytics#427]
I get an error on the second step of the installation process:
"Database Schema Installation failed. Please check the error log file for more details.". `owa-data/logs/errors` shows that there is a problem with the `owa_user` table:

```
12:55:33 2019-10-16 545 [debug_log] Starting installation of module: base 
12:55:33 2019-10-16 545 [debug_log] owa_request Table Created. 
12:55:33 2019-10-16 545 [debug_log] owa_session Table Created. 
12:55:33 2019-10-16 545 [debug_log] owa_document Table Created. 
12:55:33 2019-10-16 545 [debug_log] owa_feed_request Table Created. 
12:55:33 2019-10-16 545 [debug_log] owa_click Table Created. 
12:55:33 2019-10-16 545 [debug_log] owa_ua Table Created. 
12:55:33 2019-10-16 545 [debug_log] owa_referer Table Created. 
12:55:33 2019-10-16 545 [debug_log] owa_site Table Created. 
12:55:33 2019-10-16 545 [debug_log] owa_visitor Table Created. 
12:55:33 2019-10-16 545 [debug_log] owa_host Table Created. 
12:55:33 2019-10-16 545 [debug_log] owa_exit Table Created. 
12:55:33 2019-10-16 545 [debug_log] owa_os Table Created. 
12:55:33 2019-10-16 545 [debug_log] owa_impression Table Created. 
12:55:33 2019-10-16 545 [debug_log] owa_configuration Table Created. 
12:55:33 2019-10-16 545 [debug_log] owa_user Table Creation Failed. 
12:55:33 2019-10-16 545 [debug_log] Entity Installation Failed. 
12:55:33 2019-10-16 545 [debug_log] owa_domstream Table Created. 
12:55:33 2019-10-16 545 [debug_log] owa_action_fact Table Created. 
12:55:33 2019-10-16 545 [debug_log] owa_search_term_dim Table Created. 
12:55:33 2019-10-16 545 [debug_log] owa_ad_dim Table Created. 
12:55:33 2019-10-16 545 [debug_log] owa_source_dim Table Created. 
12:55:33 2019-10-16 545 [debug_log] owa_campaign_dim Table Created. 
12:55:33 2019-10-16 545 [debug_log] owa_location_dim Table Created. 
12:55:33 2019-10-16 545 [debug_log] owa_commerce_transaction_fact Table Created. 
12:55:33 2019-10-16 545 [debug_log] owa_commerce_line_item_fact Table Created. 
12:55:33 2019-10-16 545 [debug_log] owa_queue_item Table Created. 
12:55:33 2019-10-16 545 [debug_log] owa_site_user Table Created. 
12:55:33 2019-10-16 545 [debug_log] Installation failed. 
```
The webserver logs do not show any errors.
I think there could be a relation to #376.

OWA- Version: 1.6.2
OS: Arch Linux 5.3.6
MySQL-Version: 10.4.8-MariaDB Arch Linux
PHP-Version: PHP 7.3.10
Webserver: nginx/1.16.1

**Comment by padams** [comment]
This is fixed in master.

**Comment by ngreve** [comment]
O.k. it worked with the master branch. Thought the fix would already be included in version 1.6.2 .
